springwolf-core/src/test/java/io/github/stavshamir/springwolf/asyncapi/scanners/channels/annotation/AsyncAnnotationScannerUtilTest.java

```diff
@@ -159,6 +159,7 @@ void processMessageFromAnnotationWithAsyncMessage(Class<?> classWithOperationBin
                 .messageId("simpleFoo")
                 .name("SimpleFooPayLoad")
                 .title("Message Title")
+                .contentType("application/schema+json;version=draft-07")
```

Comment:
Should this match also the json example value generator?

Reply:
I was actually surprised about the content type. I think its the wrong here ... and should be application/json